src/main/java/com/example/book_your_seat/reservation/controller/dto/AddReservationRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.example.book_your_seat.reservation.controller.dto;
+
+import com.example.book_your_seat.reservation.domain.Reservation;
+import com.example.book_your_seat.reservation.domain.ReservationStatus;
+import com.example.book_your_seat.seat.domain.Seat;
+
+import java.util.List;
+
+public record AddReservationRequest(
```

Comment:
p4
List<Seat> 보다 List<Long>으로 seatId 만 받아 오면 어떨까요??
```suggestion
public record AddReservationRequest(
```